source/switching_to_live/index.html.md.erb

```diff
@@ -51,7 +51,20 @@ Choose which card types to accept by signing into the [GOV.UK Pay admin tool](ht
 
 If you use Worldpay, SmartPay or ePDQ, you must also make sure the same card types are selected in your PSP account.
 
-### If you use Worldpay
+### Block prepaid cards
+
+You can block users from paying with prepaid cards.
```

Comment:
We might want to include some exposition about why a service might want to do this (fraud). And maybe push them towards only blocking prepaid cards if they have a problem with them (rather than just because they can).